backend/coins/btc/account_test.go

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package btc_test
```

Comment:
@benma I guess there was a reason why this package was named `btc_test`, but I'm not sure which was it. Is it ok to move inside `btc` package?

Reply:
@benma could you take a look at this ? I can't imagine any reason why tests that test code from a certain package should be outside of that package (unit tests). but if there is/was a reason I can just undo the change in this PR.

I already addressed the other change requests from @Beerosagos

Reply:
What was the reason to change it in the first place?

Often one wants to test the API of a package, not its internals, which is why the tests are often in a separate package (`..._test`). 

It does not really matter much though, so feel free to change it. Afaik the Go stdlib tests are mostly not in external packages either, and that should be a good reference.